mongodbatlas/fw_resource_mongodbatlas_alert_configuration_test.go

```diff
@@ -398,12 +397,10 @@ func TestAccConfigRSAlertConfiguration_importPagerDuty(t *testing.T) {
 }
 
 func TestAccConfigRSAlertConfiguration_DataDog(t *testing.T) {
-	SkipTestExtCred(t) // Will skip because requires external credentials aka api key
-	SkipTest(t)        // Will force skip if enabled
 	var (
 		resourceName = "mongodbatlas_alert_configuration.test"
 		projectID    = os.Getenv("MONGODB_ATLAS_PROJECT_ID")
-		ddAPIKey     = os.Getenv("DD_API_KEY")
+		ddAPIKey     = "11111111111111111111111111111111"
```

Comment:
instead of using 11. Could you just use the value `dummykey` everywhere?

Reply:
It does run a format validation, which is why some keys have a length of 32 while other 36 with a particular format. Extracted to common variables to make this clear.